### ISSUE TITLE
[Debug  :   BepInEx] System.IO.FileNotFoundException: Could not load file or assembly 'Newtonsoft.Json ---fix?

### ISSUE THREAD

**Issue by AkiraMods** [BepInEx/BepInEx#972]
### What happened?

I know this is a universal unity support module and not specific to my game and/or mods but I'm wondering if this error (appearing on all mods and bepinex as a whole) is related to BepInEx, or the mods themselves. 
More importantly, as a user(not a dev of the mods), what can I do to fix it?

The error code section provided repeats for ALL installed mods, which worked previously.
The initialization procedure prior to this executes just fine.

I believe this means the Json.Net assembly either isn't installed, or hasn't been updated within the "project(s)". However, is the issue with BepInEx Json.Net Assembly or The mods/game Assembly. (Risk of Rain 2)
& lastly, is there anything I can do to fix this problem myself? Most of the answers I can find online are in reference to VS and NuGet, which aren't for the user, but rather the Dev. 
What are my options?

### BepInEx Distribution

Stable from GitHub

### Log outputs

```txt
[Error  :   BepInEx] Error loading [RiskOfOptions 2.8.0] : Could not load file or assembly 'Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies.
[Debug  :   BepInEx] System.IO.FileNotFoundException: Could not load file or assembly 'Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies.
File name: 'Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'
  at BepInEx.Bootstrap.Chainloader.Start () [0x00695] in <9f14e8a466704f3e8aa8ee3f7cfb543b>:0
```

### Environment

```markdown
- OS: Windows 10
- BepInEx: 5.4.21.0
- Game: Risk of Rain 2 (updated 2 days ago)
```

**Comment by jshepler** [comment]
Note the part "or one of its dependencies". Newtonsoft.Json has a dependency chain that works just fine from .net's GAC, but not for Unity games since Unity uses Mono, which won't use .net's GAC and the game doesn't include them. So while you can compile the mod using .net just fine, it won't load into the game.

I believe there are tools you can use to work down the dependency chain to see what dlls are needed and try to copy them out of the GAC into the plugins folder, but that means you'd have to distribute all those dlls (I think there's at least 10, not sure) with your mod. That's not cool

I haven't tried this, but if you install mono on your machine, mono's GAC would get installed and then the game would work. Of course, that won't help anyone else who wants to use your mod.

Basically, I think unless the game was built using Newtonsoft.Json, you're pretty much sol. You can either use Unity's built-in JsonUtility (which is really bad) or use something like https://github.com/Bunny83/SimpleJSON (harder to use but works).
